src/collection/components/CollectionOrAssignmentBlocks.tsx

```diff
@@ -0,0 +1,77 @@
+import { Container } from '@viaa/avo2-components';
+import { AssignmentBlock } from '@viaa/avo2-types/types/assignment';
+import { CollectionFragment } from '@viaa/avo2-types/types/collection';
+import React, { FC, ReactNode } from 'react';
+
+import { IconBar } from '../../shared/components';
+import { COLLECTION_OR_ASSIGNMENT_BLOCK_ICONS, CollectionBlockType } from '../collection.const';
+
+import { CollectionFragmentTypeItem, CollectionFragmentTypeText } from './index';
+
+interface CollectionOrFragmentBlocksProps {
```

Comment:
Abstraheer deze naamgeving en de onderliggende subcomponenten en verplaats deze naar `shared` ipv `collection`.

Reply:
Heb je een suggestie voor de naamgeving die nog duidelijk is?
Want we hebben ook content blocks en die zijn niet hetzelfde als een collectionFragment of een AssignmentBlock

de naam is trouwens verkeerd. Het moest zijn: CollectionOrAssignmentBlocksProps

Reply:
Ik hernoem het naar BlockList + BlockListProps + BlockItemBase + BlockItemType